ibis/pyspark/tests/test_timecontext.py

```diff
@@ -13,3 +15,172 @@ def test_table_with_timecontext(client):
     expected = table.compile().toPandas()
     expected = expected[expected.time.between(*context)]
     tm.assert_frame_equal(result, expected)
+
+
+def test_window_with_timecontext(client):
```

Comment:
Can you add docstring what is expected result? Hard to read code.

Reply:
+1